Vagrantfile

```diff
@@ -6,922 +6,19 @@
-
-  # Local Machine Hosts
-  #
-  # If the Vagrant plugin goodhosts (https://github.com/goodhosts/vagrant/) is
-  # installed, the following will automatically configure your local machine's hosts file to
-  # be aware of the domains specified below. Watch the provisioning script as you may need to
-  # enter a password for Vagrant to access your hosts file.
-  #
-  # By default, we'll include the domains set up by VVV through the vvv-hosts file
-  # located in the www/ directory and in config/config.yml.
-  #
-
-  if Vagrant.has_plugin?('vagrant-goodhosts')
-    config.goodhosts.aliases = vvv_config['hosts']
-    config.goodhosts.remove_on_suspend = true
-  elsif Vagrant.has_plugin?('vagrant-hostsmanager')
-    config.hostmanager.aliases = vvv_config['hosts']
-    config.hostmanager.enabled = true
-    config.hostmanager.manage_host = true
-    config.hostmanager.manage_guest = true
-    config.hostmanager.ignore_private_ip = false
-    config.hostmanager.include_offline = true
-  elsif Vagrant.has_plugin?('vagrant-hostsupdater')
-    # Pass the found host names to the hostsupdater plugin so it can perform magic.
-    config.hostsupdater.aliases = vvv_config['hosts']
-    config.hostsupdater.remove_on_suspend = true
-  else
-    show_check = true if %w[up halt resume suspend status provision reload].include? ARGV[0]
-    if show_check
-      puts ""
-      puts " X ! There is no hosts file vagrant plugin installed!"
-      puts " X You need the vagrant-goodhosts plugin (or HostManager/ HostsUpdater ) for domains to work in the browser"
-      puts " X Run 'vagrant plugin install --local' to fix this."
-      puts ""
-    end
-  end
-
-  # Vagrant Triggers
-  #
-  # We run various scripts on Vagrant state changes like `vagrant up`, `vagrant halt`,
-  # `vagrant suspend`, and `vagrant destroy`
-  #
-  # These scripts are run on the host machine, so we use `vagrant ssh` to tunnel back
-  # into the VM and execute things. By default, each of these scripts calls db_backup
-  # to create backups of all current databases. This can be overridden with custom
-  # scripting. See the individual files in config/homebin/ for details.
-  unless Vagrant::Util::Platform.windows?
-    if Process.uid == 0
-      config.trigger.after :all do |trigger|
-        trigger.name = 'Do not use sudo'
-        trigger.ruby do |env,machine|
-          sudo_warnings
-        end
-      end
-    end
-  end
+puts "                                              #{boldred}__ #{green}__ #{blue}__ __#{creset}                                     #{creset}"
+puts " The master branch is no longer supported!    #{boldred}\\ V#{green}\\ V#{blue}\\ V /#{creset}"
+puts " Run these commands to switch to #{purple}stable#{creset}:      #{boldred} \\_/#{green}\\_/#{blue}\\_/ #{creset}"
+puts " "
+puts " #{green}git checkout stable#{creset}"
+puts " #{green}git pull#{creset}"
```

Comment:
maybe suggest to retry again with the vagrant command too?

Reply:
we don't know what the vagrant command was, could have been as simple as `vagrant status`

Reply:
So maybe a message like "try again your command after those".
As I think that the users will just try those commands and not again the vagrant one.

Reply:
I don't see why they wouldn't retry whatever command they were previously attempting, I'm keen to make the message as short and to the point as possible to avoid confusion.

If the message isn't glaringly obvious and short enough, people will ignore it and come to slack instead. They already do this with the `vagrant plugin expunge` message when updating vagrant, they don't bother to read it, too much text